frontend/eigen_cmath/include/algebra/eigen_cmath.hpp

```diff
@@ -74,6 +74,8 @@ template <typename T>
 using polar2 = cmath::polar2<transform3<T> >;
 template <typename T>
 using cylindrical2 = cmath::cylindrical2<transform3<T> >;
+template <typename T, std::size_t ROWS, std::size_t COLS>
+using matrix = math::matrix<matrix_type, T, ROWS, COLS>;
```

Comment:
@krasznaa This is a bit broken since I used the same type from `eigen_eigen.hpp`..